### PR TITLE
Rename exclusive to double_count

### PR DESCRIPTION
**Pull request by mpark63**
## tl;dr
- removed double_count as a boolean field 
- renamed exclusive as double_count

## Motivation
The field name `exclusive` is misleading, as it is a whitelist of distribution names that CAN overlap. To avoid confusion and improve readability, what used to be `exclusive` is now called `double_count`. What used to be `double_count` is removed, as it served no purpose. 

## Changes made 
- modify distribution and fine requirement schemas 
- reflect change in requirement checking functions 
- replace exclusive and remove double_count in `majors.tsx`

### double_count?: string[]
- undefined or []: the distribution can overlap with any others
  - misleading, but saves us LOTS of typing
- ['N/A']: it may not overlap with any other distributions
- ['A', 'B']: it may overlap with distributions with the name field === 'A' or 'B'
  - in fine requirements, we use the description field 

## Testing 
Locally. No changes in functionality and no errors in console.

### CODE REVIEW

**Review by mattliu-mygit — APPROVED** [review]
Awesome, ty!
